docs/source/tutorial_lti_systems.md

```diff
@@ -331,6 +331,22 @@ fig, axs = plt.subplots(6, 2, figsize=(8, 10), sharex=True, constrained_layout=T
 _ = fom.transfer_function.bode_plot(w, ax=axs)
 ```
 
+In the Bode plot visualization, you can choose a specific one or more input-output pair
+to display its plot by setting values for both the optional arguments ```input_indices``` and/or
+```output_indices```. In case you did not pass values for both of these arguments, a
```

Comment:
```suggestion
to display its plot by setting values for both the optional arguments `input_indices` and/or
`output_indices`. In case you did not pass values for both of these arguments, a
```